advanced/lib/pycubedmini.py

```diff
@@ -125,64 +129,63 @@ def __init__(self):
             self.hardware['SDcard'] = True
             # self.new_Log() # create new log file
         except Exception as e:
-            print('[ERROR][SD Card]',e)
+            print('[ERROR][SD Card]', e)
 
         # Initialize radio #1 - UHF
         try:
-            self.radio = pycubed_rfm9x.RFM9x(self.spi, self._rf_cs, self._rf_rst, self.UHF_FREQ,rfm95pw=True)
-            self.radio.dio0=self.radio_DIO0
+            self.radio = pycubed_rfm9x.RFM9x(self.spi, self._rf_cs, self._rf_rst, self.UHF_FREQ, rfm95pw=True)
+            self.radio.dio0 = self.radio_DIO0
             self.radio.sleep()
             self.hardware['Radio'] = True
         except Exception as e:
-            print('[ERROR][RADIO]',e)
+            print('[ERROR][RADIO]', e)
 
         # Initialize IMU
         try:
-            self.IMU = bmx160.BMX160_I2C(self.i2c1,address=0x68)
+            self.IMU = bmx160.BMX160_I2C(self.i2c1, address=0x68)
             self.hardware['IMU'] = True
         except Exception as e:
             print(f'[ERROR][IMU] {e}\n\tMaybe try address=0x68?')
 
         # Initialize Sun Sensors
         try:
-            sun_yn = adafruit_tsl2561.TSL2561(self.i2c2,address=0x29) # -Y
-            sun_zn = adafruit_tsl2561.TSL2561(self.i2c2,address=0x39) # -Z
-            sun_xn = adafruit_tsl2561.TSL2561(self.i2c1,address=0x49) # -X
-
-            sun_yp = adafruit_tsl2561.TSL2561(self.i2c1,address=0x29) # +Y
-            sun_zp = adafruit_tsl2561.TSL2561(self.i2c1,address=0x39) # +Z
-            sun_xp = adafruit_tsl2561.TSL2561(self.i2c2,address=0x49) # +X
-            sun_sensors=[sun_zn,sun_yn,sun_xn,sun_yp,sun_xp]
+            sun_yn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x29)  # -Y
+            sun_zn = adafruit_tsl2561.TSL2561(self.i2c2, address=0x39)  # -Z
+            sun_xn = adafruit_tsl2561.TSL2561(self.i2c1, address=0x49)  # -X
+
+            sun_yp = adafruit_tsl2561.TSL2561(self.i2c1, address=0x29)  # +Y
+            sun_zp = adafruit_tsl2561.TSL2561(self.i2c1, address=0x39)  # +Z
+            sun_xp = adafruit_tsl2561.TSL2561(self.i2c2, address=0x49)  # +X
+            sun_sensors = [sun_zn, sun_yn, sun_xn, sun_yp, sun_xp]
```

Comment:
`sun_zp` needs to be added to this list.

Reply:
Though maybe we merge the linter and then correct this later.

Reply:
Yes I'd like to do that separately and I've made an issue for it already: #17